packages/pluginutils/rollup.config.js

```diff
@@ -4,6 +4,19 @@ import typescript from '@rollup/plugin-typescript';
 
 import pkg from './package.json';
 
+function emitModulePackageFile() {
```

Comment:
Please keep PRs scoped to one package, or use individual commits for changes to multiples. Not doing so makes it difficult to generate changelogs.

Reply:
This PR is scoped. Node resolve imports from pluginutils with named exports.

Reply:
your changes to the rollup config and package.json for `pluginutils` means we'll have to release a new version and generate a changelog. this type of commit with changes to two packages makes it difficult to accurately generate a changelog for that version. I ask to split this into it's own commit in this PR or a separate PR, especially for the future.

Reply:
https://github.com/rollup/plugins/pull/419